templates/liquid/Measure.liquid

```diff
@@ -296,6 +296,56 @@
           </td>
         </tr>
         {% endif %}
+        {% if Measure.group.exists() %}
+            <tr>
+                <th scope="row"><b>Populations: </b></th>
+                <td style="padding-left: 4px;">
+                    <table class="grid-dict">
+                        <tr><th><b>Group</b></th><th><b>Population</b></th><th><b>Description</b></th></tr>
+                        {% for group in Measure.group %}
+                        <tr>
+                            <td>{{group.id}}</td>
+                            {% for population in group.population %}
+                                <tr>
+                                    <td/>
+                                    <td>{{(population.code.first() as CodeableConcept).coding.first().display}}</td>
+                                    <td>
+                                        {% if population.description %}
+                                            {{(population.description)}}
+                                        {% endif %}
+                                    </td>
+                                 </tr>
+                            {% endfor %}
+                        </tr>
+                        {% endfor %}
+                    </table>
+                </td>
+            </tr>
+            <tr>
+                <th scope="row"><b>Population Criteria: </b></th>
+                <td style="padding-left: 4px;">
+                    <table class="grid-dict">
+                        <tr><th><b>Group</b></th><th><b>Population Criteria</b></th><th><b>Expression</b></th></tr>
+                        {% for group in Measure.group %}
+                        <tr>
+                            <td>{{group.id}}</td>
+                            {% for population in group.population %}
+                                <tr>
+                                    <td/>
+                                    <td>{{(population.code.first() as CodeableConcept).coding.first().display}}</td>
+                                    <td>
+                                        {% if population.criteria.expression%}
+                                            {{(population.criteria.expression)}}
+                                        {% endif %}
+                                    </td>
+                                 </tr>
+                            {% endfor %}
+                        </tr>
+                        {% endfor %}
+                    </table>
+                </td>
+            </tr>
+        {% endif %}
         {% if Measure.supplementalData.exists() %}
```

Comment:
But this bit, to add the population groups, should be in the logic definitions. We should be creating a section of logic definitions that are based on the groups, and the scoring type for each group controls what logic definitions we put in the list. That might be pretty challenging in Liquid, but let's try it there first. If it's too much, then we can put it in the effective data requirements support in the tooling.